101-dukes-eap-openshift/azuredeploy.parameters.json

```diff
@@ -0,0 +1,31 @@
+ {
+   "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+          "value": "GEN-UNIQUE"
+        },
+        "adminPassword": {
+          "value": "GEN-PASSWORD"
+        },
+        "rhsmUserName": {
+          "value": "mfentane@redhat.com"
+        },
+        "rhsmPassword": {
+          "value": "passw0rd!"
+        },
+        "dnsLabelPrefix": {
+            "value": "GEN-UNIQUE"
+        },
+        "eapUserName": {
+          "value": "GEN-UNIQUE"
+        },
+        "eapPassword": {
+          "value": "securestring"
+        },
+        "sshPassPhrase": {
+          "value": "GEN-PASSWORD"
+        }
+    }
+    }
```

Comment:
In order for these variables to be recognized, I think you will need to add them to azuredeploy.json as well